x-pack/plugins/security/common/licensing/index.mock.ts

```diff
@@ -7,10 +7,8 @@
 
 import { of } from 'rxjs';
 
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { LICENSE_TYPE } from '../../../licensing/server';
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import type { LicenseType } from '../../../licensing/server';
+import type { LicenseType } from '../../../licensing/common/types';
+import { LICENSE_TYPE } from '../../../licensing/common/types';
```

Comment:
NOTE for @jportner , I was able to remove the restricted paths here and I think this is where you want to import from. Also, this prevents a very large "tree import" from `security_solutions` and reduces our changes or anyone else's chances of hitting memory issues from the `licensing/server` plugin.